src/Microsoft.IdentityModel.Logging/IdentityModelEventSource.cs

```diff
@@ -84,6 +98,11 @@ public void WriteVerbose(string message)
             }
         }
 
+        /// <summary>
+        /// Writes a verbose event log without generating an event.
```

Comment:
should be same "summary" as a above. Only difference is 'formatting parameters' are passed.